src/main/java/org/folio/service/orders/OrderRolloverService.java

```diff
@@ -86,10 +111,19 @@ public Future<Void> rollover(LedgerFiscalYearRollover ledgerFYRollover, RequestC
     return fundIdsFuture
       .compose(ledgerFundIds -> configurationEntriesCache.getSystemCurrency(requestContext)
         .compose(systemCurrency -> rolloverOrdersByFundIds(ledgerFundIds, ledgerFYRollover, systemCurrency, requestContext)))
+      .recover(t -> handleOrderRolloverError(t, ledgerFYRollover, progress, requestContext))
+      .compose(aVoid -> calculateAndUpdateOverallProgressStatus(progress.withOrdersRolloverStatus(SUCCESS), requestContext))
       .onSuccess(v -> logger.info("Order Rollover success : All orders processed"))
       .onFailure(t -> logger.error("Order Rollover failed", t));
```

Comment:
`.onFailure` step will never be reached in rollover flow because of catching exception with `.recover()` method. Only `calculateAndUpdateOverallProgressStatus` results affected